new/analyse/add-package-names-to-messages.ts

```diff
@@ -0,0 +1,64 @@
+import type { Module } from '../types';
+import { dirname, join } from 'path';
+import findUp from 'find-up';
+import { readFileSync } from 'fs';
+
+import { logger } from '../helpers/logger';
+
+const { error, log } = logger('[analyse] add package names to messages');
+
+const findPackageName = (filePath: string): string => {
+  const cwd = dirname(join(process.cwd(), filePath));
```

Comment:
I'd use the variable in `env` again - to keep the single point of truth in the system.

Reply:
What variable do you mean?

Reply:
The variable that is in there, that represents the current working directory. https://github.com/xing/rlsr/blob/master/new/types.ts#L91